Update registrationClass.py

ShoniB opened the pull request:
Added functionality to check email input for a valid email address format

ShoniB committed:
Update registrationClass.py

Added functionality to check email input for a valid email address format

DermaGram reviewed (commented):
@ShoniB 
1) what does RegistrationForm do? i'm assuming this has to do w/ the signup.html? may want to add a quick descriptive note for the class.
2) i see validators.Required() in PasswordField. should this also be included for username/email?

ShoniB committed:
Update registrationClass.py

Added some comments. Required is not needed for the other inputs because there is a specified length

DermaGram reviewed (approved):
lgtm
